### PR TITLE
Fix brightness  not set with 255 value. Standard zigbee ATTR_LEVEL_MAX_LEVEL is 0xFE (254)

### PR DESCRIPTION
**Pull request by dzungpv**
Reference: https://github.com/LILCMU/WRATIOT/blob/d7c9164f1070edec13ee7a3e9bc69b458f69f179/Hardware/HopherNode/Hardware/cc2530/SourceCode/Components/stack/zcl/zcl_general.h#L413

### CODE REVIEW

File: converters/toZigbee.js [hunk 1]
```diff
@@ -351,8 +351,12 @@ const converters = {
                             {brightness: meta.state.brightness, turnedOffWithTransition: true};
                     }
 
-                    const level = state === 'off' ? 0 :
+                    let level = state === 'off' ? 0 :
                         (store[entity.deviceIeeeAddress] ? store[entity.deviceIeeeAddress].brightness : 255);
```
Comment (Koenkk) [inline converters/toZigbee.js:355]: If brightness comes from the store we can assume it's max 254 (as we control what to set in it). `store[entity.deviceIeeeAddress].brightness : 255);` should be changed to  `store[entity.deviceIeeeAddress].brightness : 254);` and the rest of the changes can be undone.

File: converters/toZigbee.js [hunk 2]
```diff
@@ -372,7 +376,11 @@ const converters = {
                          * a transition, the brightness is not recovered as it turns on with brightness 1.
                          * https://github.com/Koenkk/zigbee-herdsman-converters/issues/1073
                          */
-                        const brightness = store[entity.deviceIeeeAddress].brightness;
+                        let brightness = store[entity.deviceIeeeAddress].brightness;
+                        if (brightness === 255) {
```
Comment (Koenkk) [inline converters/toZigbee.js:380]: These changes are not needed (read comment above).

File: converters/toZigbee.js [hunk 3]
```diff
@@ -410,7 +418,10 @@ const converters = {
                 } else if (message.hasOwnProperty('brightness_percent')) {
                     brightness = Math.round(Number(message.brightness_percent) * 2.55).toString();
                 }
-
+                if (brightness === 255) {
```
Comment (Koenkk) [inline converters/toZigbee.js:421]: Can be simplified to `brightness = Math.min(254, brightness)`

Comment (dzungpv) [reply converters/toZigbee.js:421]: Yes, you can change to most simple code.

**Comment by dzungpv** [comment]
@Koenkk i rewrite it here https://github.com/Koenkk/zigbee-herdsman-converters/pull/1200

**Pull request closed without merging**